esic_fe/src/components/Loading/Loading.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import union from '../../assets/union.png'
```

Comment:
give it a better name, since we'll use the logo in the entire application, would be better to name it simply "esic_logo"

Reply:
yea sure.